File: 0263-ugly-number/0263-ugly-number.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isUgly = function(n) {
    if (n <= 0) return false;
    let cons=[2,3,5]
    for(let i of cons){
        while(n%i===0){
            n /= i

        }
    }
    return n===1
};

File: 0263-ugly-number/0263-ugly-number.py
class Solution(object):
    def isUgly(self, n):
        if (n <= 0):
            return false; 
        cons = [2, 3, 5]
        for i in cons:
            while (n % i == 0): 
                n /= i
        return n == 1;
        